vta/python/vta/vtaprogram.py

```diff
@@ -0,0 +1,64 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""VTA specific bitstream program library."""
+import os
+import sys
+import argparse
+
+def main():
+    """Main funciton"""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("target", type=str, default="",
+                        help="target")
+    parser.add_argument("bitstream", type=str, default="",
+                        help="bitstream path")
+    args = parser.parse_args()
+
+    if len(sys.argv) < 3:
```

Comment:
use [required option](https://docs.python.org/3/library/argparse.html#required) in `add_argument`, instead of counting number of args.